Changes

```diff
@@ -6,11 +6,13 @@ LIST OF CHANGES
  - An additional wr limit group - s3 - is configured.
  - The wait4path pipeline job does not have a log. To enable wr to
    recognise that these jobs are unique, echoing a random string
-   is added to the shell command. 
+   is added to the shell command.
  - Remove the limit on the number of NovaSeq runs being archived at
    the same time. Introduce a limit on the number of runs,
    irrespectively of the instrument type, which are moved to archival
    within the last hour.
+ - Persist product_release.yml file to data directory to preserve run
```

Comment:
Change 'to data directory' - 'to analysis directory'
to avoid confusion with the `Data` directory of the run folder?